lib/buildsystems/meson.rb

```diff
@@ -11,6 +11,7 @@ def self.check?(bool = true)
   end
 
   def self.build
+    puts "Additional meson_options being used: #{@meson_options}".orange
```

Comment:
```
puts "Additional meson_options being used: #{@meson_options}".orange if "#{@meson_options}" != ""
```
?

Reply:
The point is to show that `meson_options` and `cmake_options` are being seen as empty during the build.